src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -1639,6 +1648,13 @@ def test(
             keep_model = self.net
             self.net = self.ema_model.ema
 
+        context = PhaseContext(
+            criterion=self.criterion,
+            device=self.device,
+            sg_logger=self.sg_logger,
+            context_methods=self._get_context_methods(Phase.VALIDATION_BATCH_END),
```

Comment:
context_methods=self._get_context_methods(Phase.VALIDATION_BATCH_END)- I believe this is a mistake ?

Reply:
Yup, copy-paste typo